lib/dropbox_api/results/copy_batch_check_result.rb

```diff
@@ -0,0 +1,20 @@
+module DropboxApi::Results
+  class CopyBatchCheckResult < DropboxApi::Results::Base
```

Comment:
I think it makes more sense to name this `RelocationBatchV2Result` or `RelocationBatchResult` so it'd map directly with Dropbox API.

Another benefit is that it won't be too confusing when we re-use this for `move_batch`.

Reply:
Ah okay, making that change, thanks :)